invenio_rdm_records/resources/serializers/schemas.py

```diff
@@ -79,3 +86,17 @@ def get_contributors(self, obj):
         return [
             c["person_or_org"]["name"] for c in obj["metadata"].get("contributors", [])
         ] or missing
+
+    @post_dump(pass_original=True)
+    def post_dump_pipeline(self, data, original, many, **kwargs):
+        """Defines sequence of post dump steps."""
+        result = deepcopy(data)
+        for processor in self.processors:
+            if not hasattr(processor, "post_dump"):
+                raise TypeError("Post process processors must be callables")
+            # TODO does it need to be instantiated?
+            dumped_data = processor().post_dump(result, original)
```

Comment:
Question: how to avoid needing to instantiate a `processor` here? Imo, `processor().post_dump(result, original)` is uglier than `processor.post_dump(result, original)`

Reply:
I think `.processors` should be a list of already instantiated classes. If we start adding parameters to them, we'll have a hard time passing them individually as e.g. `processor_params`.